yml/microsoft/built-in/wptsextensions.yml

```diff
@@ -0,0 +1,15 @@
+---
+Name: wptsextensions.dll
+Author: k4nfr3
+Created: 2022-08-15
+Vendor: Microsoft
+VulnerableExecutables:
+  - Path: 'Task Scheduler'

```

Comment:
this would imply `%SYSTEM32%\svchost.exe` on modern Windows versions, am I right? That's also what's visible in the screenshot in the linked blog.